ipa-multipoint/ipa_multipoint_jni/Cargo.toml

```diff
@@ -21,20 +21,16 @@ authors = ["Antoine Toulme <antoine@lunar-ocean.com>"]
 repository = "https://github.com/hyperledger/besu-native"
 edition = "2018"
 
+
 [dependencies]
-verkle-spec = { git = "https://github.com/crate-crypto/rust-verkle", branch = "master" }
-verkle-trie = { git = "https://github.com/crate-crypto/rust-verkle", branch = "master" }
-ipa-multipoint = { git = "https://github.com/crate-crypto/ipa_multipoint", branch = "banderwagon_migration" }
-banderwagon = { git = "https://github.com/crate-crypto/banderwagon" }
-bandersnatch = "0.1.1"
-ark-ff = { version = "^0.3.0", default-features = false }
-ark-ec = { version = "^0.3.0", default-features = false }
-ark-serialize = { version = "^0.3.0", default-features = false }
-ark-std = { version = "^0.3.0", default-features = false }
+ipa-multipoint = { git = "https://github.com/dragan2234/rust-verkle" }
+banderwagon = { git = "https://github.com/dragan2234/rust-verkle" }
+ark-ff = { version = "^0.4.0", default-features = false }
+ark-serialize = { version = "^0.4.2", default-features = false }
```

Comment:
You should be able to avoid these imports by doing `use bandwagon::trait_defs::*`

They just reexport the necessary components from ark-ff and ark-serialize, so that arkworks doesn't leak into dependents